oidc_provider/lib/errors.py

```diff
@@ -7,7 +7,8 @@
 class RedirectUriError(Exception):
 
     error = 'Redirect URI Error'
-    description = 'The request fails due to a missing, invalid, or mismatching redirection URI (redirect_uri).'
+    description = 'The request fails due to a missing, invalid, or mismatching' \
+                  ' redirection URI (redirect_uri).'
```

Comment:
```suggestion
                  ' redirection URI (redirect_uri).')
```

Reply:
Discussed elsewhere. Going to hold off on formatting changes as this is a fork and it may make it harder to maintain from upstream later. Unless anyone else feels strongly enough we should make any of these changes.